handbook/communication/company_meeting.md

```diff
@@ -1,6 +1,6 @@
 # Company meeting
 
-1. Schedule: every Monday 10:30-11:00am PST/PDT (link in the calendar event)
+1. Schedule: every Monday 8:00-8:30am PST/PDT (link in the calendar event)
    - If Monday is a holiday, the meeting is held on the next non-holiday day.
```

Comment:
I believe we don't respect that anymore, since holidays aren't global?

Reply:
There are a small number where it is documented that we move company meeting.

```suggestion
   - If Monday is a [holiday](../people-ops/holidays.md), the meeting is held on the next non-holiday day.
```